vivisect/analysis/__init__.py

```diff
@@ -66,6 +67,9 @@ def addAnalysisModules(vw):
 
     elif fmt == 'elf':  # ELF ########################################################
 
+        # elfplt wants to be run before generic.entrypoints.
```

Comment:
no, this is so that the elfplt code itself runs before other codeflow analysis gets done, which would likely cause the PLT entries to be analyzed beforehand.  just looking for consistent PLT analysis over fresh code.

Reply:
:+1: